lms/djangoapps/verify_student/models.py

```diff
@@ -42,9 +42,8 @@
     rsa_decrypt,
     rsa_encrypt
 )
+from openedx.core.djangoapps.signals.signals import LEARNER_SSO_VERIFIED
 from openedx.core.storage import get_storage
-from openedx_events.learning.signals import IDV_ATTEMPT_APPROVED
```

Comment:
I know this is independent of your changes here, but this removal got me thinking about whether it's enough for this event to be emitted from the [API](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/verify_student/api.py#L57). I'm wondering if we should also emit these events when the model is saved to ensure that these events are logged when the instances are modified a different way (e.g. via the admin). Do you have any thoughts about that since you've worked so much with the signals and handlers in this app?

Reply:
Good call, I also saw this and do think it's problematic. We should be firing events like this as a result of the model changing. Maybe not needed to release but we should clean it up before the inevitable bug/confusion when this record gets saved by another method